ome-demoserver.yml

```diff
@@ -126,10 +126,13 @@
       omero_server_dbpassword: "{{ vault.omero_server_dbpassword }}"
       omero_server_rootpassword: "{{ vault.omero_server_rootpassword }}"
       omero_server_systemd_limit_nofile: 16384
+      # Once omego version bug is fixed, we can remove this hard-coding.
```

Comment:
Perhaps add a `TODO`

Reply:
Do you mean something more than just adding TODO do the comment? (a la --Fixes in github?) e.g. something for SCC?

Reply:
Perfect. Thanks.